src/gpu/nvidia/sycl_cuda_engine.hpp

```diff
@@ -78,16 +78,12 @@ class sycl_cuda_engine_t : public dnnl::impl::sycl::sycl_engine_base_t {
     CUdevice get_underlying_device() const;
     cudnnHandle_t *get_cudnn_handle();
     cublasHandle_t *get_cublas_handle();
-    const bool has_primary_context() const { return primary_context_; }
```

Comment:
@densamoilov Am I allowed to remove this public member function? I could change this just to return `true` instead

Reply:
If you think it's not needed feel free to remove it